app/controllers/idv/in_person/address_search_controller.rb

```diff
@@ -22,10 +22,6 @@ def addresses
       def geocoder
         @geocoder ||= ArcgisApi::Geocoder.new
       end
-
-      def permitted_params
```

Comment:
is this removed b/c we're expanding the locations to proof at and therefore all addresses are permitted?

Reply:
I'm no Rails expert, but if we're individually selecting the parameter attribute as we are with `params[:address]`, I think it should be fine, since we're essentially ignoring the rest of what might exist in `params` anyways.